### ISSUE TITLE
Can't use Common Crawl Downloader in package downloaded from PyPI

### ISSUE THREAD

**Issue by BMarcin** [BMarcin/MordinezNLP#10]
# Can't use Common Crawl Downloader in package downloaded from PyPI

Used code
```python
from MordinezNLP.downloaders import CommonCrawlDownloader

ccd = CommonCrawlDownloader(
    [
        "reddit.com/r/space/*",
        "reddit.com/r/spacex/*",
    ]
)
ccd.download('./test_data')
```

Error log
```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-5-43bb9781abb4> in <module>
      7     ]
      8 )
----> 9 ccd.download('./test_data')

/opt/venv/lib/python3.7/site-packages/MordinezNLP/downloaders/CommonCrawlDownloader.py in download(self, save_to, base_url, sleep_time)
    150             custom_headers=[entry['headers'] for entry in entries_to_download],
    151             streamable=repeat(True),
--> 152             sleep_time=sleep_time
    153         )
    154 

/opt/venv/lib/python3.7/site-packages/MordinezNLP/downloaders/Basic.py in download_urls(urls, file_type_handler, threads, sleep_time, custom_headers, streamable, max_retries)
     64                 )
     65             )
---> 66             return [file_type_handler(item) for item in downloaded_strings]
     67 
     68     @staticmethod

/opt/venv/lib/python3.7/site-packages/MordinezNLP/downloaders/Basic.py in <listcomp>(.0)
     64                 )
     65             )
---> 66             return [file_type_handler(item) for item in downloaded_strings]
     67 
     68     @staticmethod

/opt/venv/lib/python3.7/site-packages/MordinezNLP/downloaders/CommonCrawlDownloader.py in _common_crawl_gzip_to_text_processor(data_in)
    104             Source html of crawled site.
    105         """
--> 106         gzipped_data = gzip_to_text_data_processor(data_in)
    107         return gzipped_data.strip().split("\n\n", 2)[2]
    108 

NameError: name 'gzip_to_text_data_processor' is not defined
```

**Comment by BMarcin** [comment]
Fixed in [0.1.0-a2](https://github.com/BMarcin/MordinezNLP/tree/0.1.0-a2)

**Comment by BMarcin** [comment]
Fixed in 0.1.0-a3
